mrtt-ui/package.json

```diff
@@ -32,11 +32,11 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test  --passWithNoTests",
```

Comment:
This feels better than having a commented out test to make it not fail because of _no_ tests.

Reply:
nice!

Reply:
We might consider removing it later when there are actual tests. It makes sense to fail if we are expecting tests, and none get loaded (maybe because of a directory restructure, or whatever)